package.json

```diff
@@ -29,15 +29,16 @@
     "devDependencies": {
         "@intlify/vue-i18n-loader": "^4.1.0",
         "@vue/cli-plugin-babel": "^4.5.15",
-        "@vue/cli-plugin-eslint": "^4.5.15",
+        "@vue/cli-plugin-eslint": "~4.5.0",
         "@vue/cli-plugin-pwa": "^4.5.15",
         "@vue/cli-service": "^4.5.15",
         "@vue/compiler-sfc": "3.2.26",
+        "@vue/eslint-config-prettier": "^6.0.0",
         "babel-eslint": "^10.1.0",
-        "eslint": "^7.32.0",
-        "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-prettier": "^4.0.0",
-        "eslint-plugin-vue": "^7.20.0",
+        "eslint": "^6.7.2",
+        "eslint-plugin-prettier": "^3.3.1",
+        "eslint-plugin-vue": "^7.0.0",
+        "prettier": "^2.2.1",
```

Comment:
Why are we downgrading some dependencies?
eg: eslint `7.32.0` -> `6.7.2`

Reply:
npm and pnpm couldn't resolve dependencies when I first tried fetching packages, due to `@vue/cli-plugin-eslint@4.5.15` having set `eslint@">= 1.6.0 < 7.0.0"` as a peer dependency. Yarn doesn't actually care, so I can as well revert this commit. Especially since when will have switched to vite, this dependency will no longer be there.

Would you prefer a commit reverting the changes, or to leave it out during a rebase? The latter will require a force push, but that won't have any implications beyond this PR.

Reply:
A revert works just as fine :)